enarx-keep-sgx-shim/src/main.rs

```diff
@@ -14,6 +14,18 @@ fn panic(_info: &core::panic::PanicInfo) -> ! {
     loop {}
 }
 
+#[cfg(not(test))]
+#[no_mangle]
+extern "C" fn _Unwind_Resume() {
+    loop {}
+}
+
+#[cfg(not(test))]
+#[no_mangle]
+pub extern "C" fn rust_eh_personality() {
+    loop {}
+}
```

Comment:
Can you give a brief comment as to what these are and why they are here?

Reply:
(A comment in the code for future readers, that is...)

Reply:
yes